src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -456,17 +454,19 @@ def initialize(wrapped)
     # SingleReqView limits access to an ActiveCall's methods for use in server
     # handlers that receive just one request.
     SingleReqView = view_class(:cancelled, :deadline, :metadata,
```

Comment:
Thought that SingleReqView and MultiReqView were only for call objects in server handlers, in which case trailing metadata shouldn't be relevant?